packages/azure-kusto-ingest/src/resourceManager.ts

```diff
@@ -23,11 +25,12 @@ export class IngestClientResources {
         readonly securedReadyForAggregationQueues: ResourceURI[] | null = null,
         readonly failedIngestionsQueues: ResourceURI[] | null = null,
         readonly successfulIngestionsQueues: ResourceURI[] | null = null,
-        readonly containers: ResourceURI[] | null = null
+        readonly containers: ResourceURI[] | null = null,
+        readonly statusTable: ResourceURI[] | null = null
```

Comment:
statustable is singular - make it a singular ResourceURI and check at ctor?

Reply:
We don't want to commit to it not being a list

Reply:
then call it statusTables?